eZ/Publish/Core/REST/Server/Output/ValueObjectVisitor/PublishedRole.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * File containing the PublishedRole ValueObjectVisitor class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace eZ\Publish\Core\REST\Server\Output\ValueObjectVisitor;
+
+use eZ\Publish\Core\REST\Common\Output\Generator;
+use eZ\Publish\Core\REST\Common\Output\Visitor;
+
+/**
+ * PublishedRole value object visitor.
+ *
+ * @todo coverage add unit test
```

Comment:
Yes, I know :) This is based on existing visitors (CreatedRole et al) and since they are so similar I'd like to do coverage for them separately in a followup.